### PR TITLE
miscellaneous minor improvements

### PR DESCRIPTION
**Pull request by romain-dartigues**

### CODE REVIEW

File: docker_leash/checks/base.py [hunk 1]
```diff
@@ -1,18 +1,25 @@
 # vim:set ts=4 sw=4 et:
+'''
+Base
+----
```
Comment (kumy) [inline docker_leash/checks/base.py]: Dashes or equals signs?

File: docker_leash/checks/deny.py [hunk 2]
```diff
@@ -1,17 +1,23 @@
 # vim:set ts=4 sw=4 et:
 
+'''
+Deny
+----
```
Comment (kumy) [inline docker_leash/checks/deny.py]: "equals"?

File: docker_leash/leash_server.py [hunk 3]
```diff
@@ -129,44 +142,47 @@ def authz_request():
 
     try:
         processor.run(request.data or {})
-    except UnauthorizedException as e:
-        print "REQUEST DENIED: %s\n" % str(e)
+    except UnauthorizedException as error:
+        print "REQUEST DENIED: %s\n" % str(error)
```
Comment (kumy) [inline docker_leash/leash_server.py]: We should this of adding a logger...

**Comment by romain-dartigues** [comment]
I'm leaving you in charge.

File: docs/index.rst [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+   * - Builds and tests coverage
+     - .. image:: https://codecov.io/gh/docker-leash/leash-server/branch/master/graph/badge.svg
+          :target: https://codecov.io/gh/docker-leash/leash-server
+
+       .. image:: https://travis-ci.org/docker-leash/leash-server.svg?branch=master
+          :target: https://travis-ci.org/docker-leash/leash-server
+
+       .. image:: https://api.codacy.com/project/badge/Grade/444467f3204246318ddc8a1af5af89bc
+          :target: https://www.codacy.com/app/docker-leash/leash-server?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=docker-leash/leash-server&amp;utm_campaign=Badge_Grade
+
```
Comment (kumy) [inline docs/index.rst]: Didn't saw how it looks like, but I already love it!!!

File: docker_leash/checks/base.py (hunk 1 above)
Comment (romain-dartigues) [reply docker_leash/checks/base.py]: Well, it's a level under checks, so:
```
Docker leash
############

Checks
======

Base
----
```

That was the tough.

Comment (kumy) [reply docker_leash/checks/base.py]: I wasn't aware of it ;)

**Comment by romain-dartigues** [comment]
Hm, BTW, `pylint` reported a more "serious" error I haven't fixed yet, maybe you could look at it:

```
************* Module docker_leash.checks.base
R:  1, 0: Cyclic import (docker_leash.leash_server -> docker_leash.processor) (cyclic-import)
```

**Comment by kumy** [comment]
I think it's a forget while moving `app` to `__init__.py`

This should fix it:
```
diff --git a/docker_leash/processor.py b/docker_leash/processor.py
index 4170c78..95dc5c7 100644
--- a/docker_leash/processor.py
+++ b/docker_leash/processor.py
@@ -4,11 +4,11 @@ import json
 
 import yaml
 
-from . import checks
+from . import checks, app
 from .action_mapper import ActionMapper
 from .config import Config
 from .exceptions import NoSuchCheckModuleException, UnauthorizedException
-from .leash_server import app
+# from .leash_server import app
 from .payload import Payload
 
 
```

**Comment by kumy** [comment]
LGTM

**Pull request merged**